### ISSUE TITLE
Ethiopia: Woreda and zone update

### ISSUE THREAD

**Issue by NateHeard** [ICPI/GIS#109]
This ticket will track an update to the Ethiopia woreda boundaries in the shapefile Ethiopia_PROD_6_Woreda_WoredaLsib_2020_Apr. The update will demarcate parts of Addis Ababa (Yeka Woreda 14, Akaki Kality Woredas 12 & 13) using reference files provided by the country team.

**Comment by NateHeard** [comment]
Using sketches and tabular updates provided by the country team , I've edited the woreda level as follows.

**Akaki Kality woreda**
![Akaki Kality Update](https://user-images.githubusercontent.com/13320367/97207315-44af4a00-1790-11eb-9448-757977bf866d.png)

**Yeak woreda**
![Yeka Update](https://user-images.githubusercontent.com/13320367/97207324-45e07700-1790-11eb-9c85-74c3ab92daf4.png).

The proposed changes are now with the country team for review.

**Comment by NateHeard** [comment]
DATIM has recently been updated to incorporate changes in  administrative hierarchies.

- We need to make corresponding changes to the maps.	
- These changes affect the current regional shape file, two zone shape files and 22 woreda shape files		
- One region,  36 zone and 12 woreda shape files will newly be formed (in addition to revision of one zone and 14 woreda borders)	
- The borders of one zone and 14 woredas will also be revised

**Comment by NateHeard** [comment]
I complete a draft of the woreda boundaries, which increases the total number of mappable units from 1,041 to 1,063. To quality check my work, I attached a DATIM hierarchy snapshot to the updated shapefile using the uid to confirm a clean join. Then I checked that the names in DATIM aligned perfectly with the names in the shapefile. The file is with the country team for review. I sent the shapefile and the tabular data saved as an xls to facilitate their review.
